src/schemas/__init__.py

```
@@ -1,9 +1,9 @@
 from schemas.movies import (
     MovieDetailSchema,
     MovieListResponseSchema,
-    MovieListItemSchema,
     MovieCreateSchema,
-    MovieUpdateSchema
+    MovieUpdateSchema,
```

Ruff (0.11.9)

`schemas.movies.MovieUpdateSchema` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)